src/sections/downloads/linux-debian-template.marko

```diff
@@ -23,9 +23,9 @@ import ManualTemplate from './linux-manual-template.marko';
             If you need to correct these after installation, please re-run the installation script.
         </alert>
         <h5>curl</h5>
-        <pre><code class="bash">sudo curl -o- https://raw.githubusercontent.com/Sonarr/Sonarr/develop/distribution/debian/install.sh | bash</code></pre>
+        <pre><code class="bash">curl -o install.sh https://raw.githubusercontent.com/Sonarr/Sonarr/develop/distribution/debian/install.sh && sudo bash install.sh</code></pre>
```

Comment:
Should we also cleanup the install.sh after?

Is there a switch to overwrite the file if it already exists we can use to prevent it from asking the user and breaking things?

Reply:
I wouldn't clean the file up, in case the user needs to re-run it. They can remove it if they like. It is a shame that we can't just `| sudo bash` it, why people putting bash weird places!

Would a shebang `#!/usr/bin/env bash` work better?

Reply:
> Should we also cleanup the install.sh after?

I'm not sure we want to clean it up either in case user wants to then run it themselves. Should we be naming it installSonarr.sh perhaps so its more clear on their FS?

> Is there a switch to overwrite the file if it already exists we can use to prevent it from asking the user and breaking things?

`curl -o` and `wget -O` both already overwrite the the file if it exists.

>It is a shame that we can't just `| sudo bash` it, why people putting bash weird places! Would a shebang `#!/usr/bin/env bash` work better? This is a common, more compatible way to shebang python scripts.

Its not about bash being in an odd place that it currently doesn't work, its that stdin is already read during the pipe so any more user input isn't being requested by the script. Its then not parsing variables properly.

Reply:
would suggest renaming to install-sonarr

Reply:
Looks like we might be able to use `/dev/tty` to read the input instead, opened a PR for that: https://github.com/Sonarr/Sonarr/pull/6299